# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/ron/src/lib.rs
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 third_party/rust/ron/src/lib.rs
--- a/third_party/rust/ron/src/lib.rs	Fri Oct 03 18:01:30 2025 +0000
+++ b/third_party/rust/ron/src/lib.rs	Fri Oct 03 18:08:29 2025 +0000
@@ -12,8 +12,17 @@
 #![deny(clippy::unreachable)]
 #![deny(unsafe_code)]
 #![allow(clippy::missing_errors_doc)] // FIXME
+#![warn(clippy::alloc_instead_of_core)]
+#![warn(clippy::std_instead_of_alloc)]
+#![warn(clippy::std_instead_of_core)]
 #![doc = include_str!("../README.md")]
-#![doc(html_root_url = "https://docs.rs/ron/0.10.1")]
+#![doc(html_root_url = "https://docs.rs/ron/0.11.0")]
+#![no_std]
+
+#[cfg(feature = "std")]
+extern crate std;
+
+extern crate alloc;
 
 pub mod de;
 pub mod ser;
@@ -24,6 +33,7 @@
 pub mod extensions;
 
 pub mod options;
+pub mod util;
 
 pub use de::{from_str, Deserializer};
 pub use error::{Error, Result};